pow/src/main/java/tech/pegasys/teku/pow/Eth1StatusLogger.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.pow;
+
+import static tech.pegasys.teku.logging.StatusLogger.STATUS_LOG;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class Eth1StatusLogger {
+  private static final int LOG_INTERVAL = 30000;
+  private final AtomicBoolean timerActive = new AtomicBoolean(false);
+  private Instant startInstant;
+  private ScheduledExecutorService executor;
+  private ScheduledFuture<?> scheduledFuture;
+
+  private void start() {
+    this.startInstant = Instant.now();
+    this.timerActive.set(true);
+    this.executor = Executors.newScheduledThreadPool(1);
+    scheduledFuture =
+        executor.scheduleAtFixedRate(
+            () ->
+                STATUS_LOG.eth1ServiceDown(
+                    Duration.between(startInstant, Instant.now()).getSeconds()),
+            LOG_INTERVAL,
+            LOG_INTERVAL,
+            TimeUnit.MILLISECONDS);
+  }
```

Comment:
Sorry, I led you astray here - don't know why I didn't think of this when I mentioned `ScheduledExecutorService`.  We have an abstraction (`AsyncRunner`) that makes running things after some period a lot more testable and generally makes managing life cycle easier.  And we also have one for accessing time - `TimeProvider` (which you can get from the `ServiceConfig` passed into `PowchainService` constructor).

So using those I think we wind up with:
```
public class Eth1StatusLogger {
  private static final Logger LOG = LogManager.getLogger();
  private static final int LOG_INTERVAL = 30000;

  private final AsyncRunner asyncRunner;
  private final TimeProvider timeProvider;

  private Optional<Cancellable> activeReporter = Optional.empty();

  public Eth1StatusLogger(final AsyncRunner asyncRunner, final TimeProvider timeProvider) {
    this.asyncRunner = asyncRunner;
    this.timeProvider = timeProvider;
  }

  synchronized void success() {
    activeReporter.ifPresent(
        reporter -> {
          reporter.cancel();
          activeReporter = Optional.empty();
        });
  }

  synchronized void fail() {
    if (activeReporter.isEmpty()) {
      final UnsignedLong outageStartInSeconds = timeProvider.getTimeInSeconds();
      final Cancellable reporter =
          asyncRunner.runWithFixedDelay(
              () -> reportOutage(outageStartInSeconds),
              LOG_INTERVAL,
              TimeUnit.MILLISECONDS,
              error -> LOG.error("Failed to check ETH1 status", error));
      activeReporter = Optional.of(reporter);
    }
  }

  private void reportOutage(final UnsignedLong outageStartInSeconds) {
    STATUS_LOG.eth1ServiceDown(timeProvider.getTimeInSeconds().minus(outageStartInSeconds));
  }
}
```

So basically when a failure comes in, start a periodic report of how long the failure has been happening and when success happens cancel that reporting.